docs/API.md

```diff
@@ -87,7 +87,8 @@ A function used to convert a query string into an object that gets passed to rou
 While the router is matching, errors may bubble up, here is your opportunity to catch and deal with them. Typically these will come from async features like [`route.getComponents`](#getcomponentslocation-callback), [`route.getIndexRoute`](#getindexroutelocation-callback), and [`route.getChildRoutes`](#getchildrouteslocation-callback).
 
 ##### `onUpdate()`
-Called whenever the router updates its state in response to URL changes.
+Called whenever the router updates its state in response to URL changes. 
+Within this function the current location is accessible through `this.state.location.pathname`.
```

Comment:
This should be just `this.state.location`. `pathname` won't have things like query variables included.